layers/+lang/python/packages.el

```diff
@@ -330,3 +331,12 @@ fix this issue."
   (spacemacs|use-package-add-hook xcscope
     :post-init
     (spacemacs/setup-helm-cscope 'python-mode)))
+
+(defun python/init-py-isort ()
+  (use-package py-isort
+    :defer t
```

Comment:
Use `:if python-enable-sort-imports-on-save` to load the package if the variable is set